emoji/src/main/java/com/vanniktech/emoji/Utils.java

```diff
@@ -106,4 +106,8 @@ static void fixPopupLocation(@NonNull final PopupWindow popupWindow, @NonNull fi
   private Utils() {
     throw new AssertionError("No instances.");
   }
+
+  static boolean isEmpty(@Nullable String text) {
```

Comment:
I can use that indeed, just decided not to so I don't have to use Robolectric for the test. As you prefer :)